shortcut_composer/api_krita/wrappers/node.py

```diff
@@ -34,6 +35,18 @@ class Node():
 
     node: KritaNode
 
+    def add_child_node(self, child: 'Node', above: 'Node') -> bool:
+        """
+        Adds the given node in the list of children.
```

Comment:
PEP257 wymaga formy rozkazującej
```suggestion
        Add the given node in the list of children.
```